Source/WTF/wtf/DataLog.cpp

```diff
@@ -139,10 +138,10 @@ void setDataFile(const char* path)
     const char* pathToOpen = path;
 
     if (path) {
-        const char* pidFormat = strstr(path, "%pid");
-        if (pidFormat) {
-            size_t leadingPathLength = pidFormat - path;
-            size_t pathCharactersAvailable = std::min(maxPathLength, leadingPathLength);
+        auto spanSpan = span(path);
```

Comment:
Did you mean to name this `pathSpan`? Also, the `span` function that takes a `char*` needs the name `unsafeSpan` at Geoff's suggestion.

Reply:
Yes, `pathSpan` was the intention :)